packages/bpk-component-inset-banner/src/BpkInsetBanner.tsx

```diff
@@ -0,0 +1,114 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { cssModules } from '../../bpk-react-utils';
+
+import STYLES from './BpkInsetBanner.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+type Props = {
+  title?: string;
+  subheadline?: string;
+  logo?: string;
+  backgroundColor: string;
+  variant: 'onLight' | 'onDark';
+  body?: {
+    text: string;
+    link?: string;
+    linkText?: string;
+  };
+  // [TO-DO] In future iteration we will add a button to CTA section with popover functionality
+  callToAction?: {
+    text?: string;
+  };
+};
+
+const BpkInsetBanner = ({
+  backgroundColor,
+  body,
+  callToAction,
+  logo,
+  subheadline,
+  title,
+  variant,
+}: Props) => {
+  const classNames = [];
+  if (variant === 'onLight') {
+    classNames.push('bpk-inset-banner--on-light');
+  }
+  if (variant === 'onDark') {
+    classNames.push('bpk-inset-banner--on-dark');
+  }
+  if (body) {
+    classNames.push('bpk-inset-banner--with-body');
+  }
```

Comment:
With the above suggestion for Variants, this could then become
 
```suggestion
  const classNames = getClassName(
      'bpk-inset-banner',
      variant && `bpk-inset-banner--${variant}`
      body && 'bpk-inset-banner--with-body'
  );
```

Then on L64 you can just do

`className={classNames}`